controllers/grid/files/SelectableSubmissionFileListCategoryGridHandler.inc.php

```diff
@@ -42,7 +42,7 @@ class SelectableSubmissionFileListCategoryGridHandler extends CategoryGridHandle
 	 * @param $capabilities integer A bit map with zero or more
 	 *  FILE_GRID_* capabilities set.
 	 */
-	function SelectableSubmissionFileListCategoryGridHandler($dataProvider, $stageId, $capabilities) {
+	function SelectableSubmissionFileListCategoryGridHandler($dataProvider, $stageId, $capabilities = 0) {
```

Comment:
It is already set a default value for capabilities in the FileGridCapabilities... do we need this one also?